package.json

```diff
@@ -14,6 +14,7 @@
     "i18next-xhr-backend": "^1.5.1",
     "js-cookie": "^2.2.0",
     "lodash.get": "^4.4.2",
+    "lodash": "^4.17.10",
```

Comment:
Do we really need to bring in lodash?

Reply:
@dlabaj personal preference for utility funcs, I ll remove it.